modules/svg/hoot.js

```diff
@@ -81,11 +170,7 @@ export function svgHoot(projection, context, dispatch) {
     drawHoot.geojson = function(gj) {
         if (!arguments.length) return svgHoot.geojson;
 
-        if ( _isArray( gj ) ) {
-            svgHoot.geojson = svgHoot.geojson.concat( gj );
-        } else {
-            svgHoot.geojson.push( gj );
-        }
+        svgHoot.geojson = gj;
```

Comment:
when i was trying to see where we called this geojson() function i found once case in [layerManager](https://github.com/ngageoint/hootenanny-ui/blob/hoot2x/modules/Hoot/managers/layerManager.js#L332-L339).

I used that to determine 'we should make the geojson here a feature collection'. maybe instead it should be 'we should just keep an array' and change the linked invocation to take an array.